test/integration/test_liquid_market_mock.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+from hummingbot.market.liquid.liquid_api_order_book_data_source import LiquidAPIOrderBookDataSource
```

Comment:
Don't import anything before the `sys.path.insert()` line... you cannot access the `hummingbot` module here yet.